### PR TITLE
NFT updates

### PR DESCRIPTION
**Pull request by Tschakki**
Updates and improves the NFT app for the tutorial.

Adds a new transaction asset to transfer NFTs.

### CODE REVIEW

File: tutorials/nft/blockchain-app/index.js [hunk 1]
```diff
@@ -8,8 +8,8 @@ const {
 } = require('lisk-sdk');
 
 // 2.Import NFT module and Plugin
-const { NFTModule } = require('./nft_module');
-const { NFTAPIPlugin } = require('./nft_api_plugin');
+const { NFTModule } = require('./nft-module');
+const { NFTAPIPlugin } = require('./nft-api-plugin');
```
Comment (nazarhussain) [inline tutorials/nft/blockchain-app/index.js]: We tend to enforce underscore file names in all our projects and repos. `lisk-core`, `lisk-sdk`. Why we want to introduce hyphen file format here?

File: tutorials/nft/blockchain-app/nft-api-plugin/db.js [hunk 2]
```diff
@@ -81,16 +81,35 @@ const getNFTHistory = async (db, dbKey) => {
...
     const module = registeredModules.find(m => m.id === trx.moduleID);
     if (module.name === 'nft') {
-      const dbKey = `${trx.asset.name}`;
-      const base32Address = cryptography.getBase32AddressFromPublicKey(Buffer.from(trx.senderPublicKey, 'hex'), 'lsk');
-      const savedHistory = await getNFTHistory(db, dbKey);
-      const nftHistory = [Buffer.from(base32Address, 'binary'), ...savedHistory];
-      const encodedNFTHistory = codec.encode(encodedNFTHistorySchema, { nftHistory });
-      await db.put(dbKey, encodedNFTHistory);
+    let dbKey, savedHistory, base32Address, nftHistory, encodedNFTHistory;
+      if (trx.assetID === 0){
```
Comment (nazarhussain) [inline tutorials/nft/blockchain-app/nft-api-plugin/db.js]: Please create some constant for such values, e.g.  `0` here.

**Review by nazarhussain — CHANGES_REQUESTED** [review]

File: tutorials/nft/blockchain-app/index.js (hunk 1 above)
Comment (Tschakki) [reply tutorials/nft/blockchain-app/index.js]: I wasn't aware of this standard. If you feel it is needed, I can change it back to underscore.

Comment (nazarhussain) [reply tutorials/nft/blockchain-app/index.js]: In start of the Lisk-Core some files were camelcase and we agreed to stick with underscore file names. So why if you look now in `lisk-sdk` and `lisk-core` you will notice all files in camel case. 

I am not sure if you need to change it now or not. But would prefer if we keep it same in all our source code no matter which project or repo it is.

**Review by nazarhussain — APPROVED** [review]

**Pull request merged**